source/intro_release_notes/release_notes_enterprise/resolved_issues_6102.rst

```diff
@@ -15,3 +15,4 @@ The following issues has been solved in 6.10.2:
 - `Fix PyOne installation through pip <https://github.com/OpenNebula/one/issues/6784>`__.
 - `Fix the list of attibutes that can be overriden in vmm_exec_kvm.conf <https://github.com/OpenNebula/one/issues/6548>`__.
 - `Fix a rare crash in 'onedb fsck' caused by a locked MarketPlaceApp in a federated environment <https://github.com/OpenNebula/one/issues/6793>`__.
+- `Fix [FSunstone] filter VMs on "Locked" gives empty white page <https://github.com/OpenNebula/one/issues/6768>`__.
```

Comment:
we shoulnd't be using FSunstone anylonger, now is just Sunstone. Also we don't need the brackets :)

Reply:
Fixed!